### DISCUSSION TITLE
The docs missing content

### DISCUSSION THREAD

**Discussion by yknevenky** [Sunbird-RC/community#886] [Issues]
The docs miss content in the verifiable credentials section.

![Screenshot 2023-12-14 at 3 33 48 PM](https://github.com/Sunbird-RC/community/assets/42597435/40786d14-b2a1-4b79-8239-839c2d0ee46b)

**Comment by manukalia29** [comment]
@yknevenky We have resolved the above mentioned issue. Please check the latest documentation [here](https://rc.sunbird.org/learn/readme)
